bgpd/bgp_zebra.c

```diff
@@ -1541,7 +1544,23 @@ bgp_zebra_announce_actual(struct bgp_dest *dest, struct bgp_path_info *info,
 		return ZCLIENT_SEND_SUCCESS;
 	}
 
-	/* Make Zebra API structure. */
+	/* Make Zebra API structure without zeroing large arrays. */
+	api.message = 0;
```

Comment:
If this is the case... Then we need a comment here to DO NOT FORGET to add NEW fields if added here too...